core/ui/PageControls/manager.tid

```diff
@@ -4,17 +4,16 @@ caption: {{$:/core/images/list}} {{$:/language/Buttons/Manager/Caption}}
 description: {{$:/language/Buttons/Manager/Hint}}
 
 \whitespace trim
-\define manager-button(class)
-\whitespace trim
-<$button to="$:/Manager" tooltip={{$:/language/Buttons/Manager/Hint}} aria-label={{$:/language/Buttons/Manager/Caption}} class="""$(tv-config-toolbar-class)$ $class$""">
-<$list filter="[<tv-config-toolbar-icons>match[yes]]">
+\procedure manager-button(class)
+<$button to="$:/Manager" tooltip={{$:/language/Buttons/Manager/Hint}} aria-label={{$:/language/Buttons/Manager/Caption}} class=`$(tv-config-toolbar-class)$ $(class)$`>
+<% if [<tv-config-toolbar-icons>match[yes]] %>
```

Comment:
It's a small thing, but in the core code I've written, I have been omitting the whitespace between `<%` and `if`, and within `<%else%>` and `<%endif%>`. Now that I check, I see that both styles have crept into the core. I favour the shorter form, which I think is also more consistent with the way that we write widget invocations.

Perhaps this is something that we should come back to in a separate PR and perhaps some wikitext authoring style guidelines for the core. Consistency is important and so we should try to address it.

Reply:
Here I do prefer readability over space. So I would prefer `<% if [[filter]] %>` -- I think the extras spaces here are much more readable. Especially for `<%elseif%><%else%>` the differences are hard to see